helpers/result_to_json.cr

```diff
@@ -27,8 +27,9 @@ class TestCase
   property status : String?
   property message : String?
   property output : String?
+  property task_id : Int32?
 
-  def initialize(@name, @test_code, @status, @message, @output)
+  def initialize(@name, @test_code, @status, @message, @output, @task_id = 1)
```

Comment:
Shouldn't this default to `nil`? Most of the exercises won't have task IDs, as they're only used in concept exercises and I don't think Crystal has any.